Podfile

```diff
@@ -7,4 +7,6 @@ target 'StockCount' do
   pod 'Realm', '~>3.13', :modular_headers => true
   pod 'RealmSwift', '~>3.13'
   pod 'SKTCapture', '~>1.1'
+  pod 'Alamofire', '= 4.8.2'
+  pod 'SwiftyJSON'
```

Comment:
We should specify a version for `SwiftyJSON`

Reply:
- will fix on barcode consolidation PR because there are warning issue on those PR that is related this.